src/components/flowchart/draw.js

```diff
@@ -153,6 +153,7 @@ export const drawNodes = function (changed) {
     enterNodes
       .attr('tabindex', '0')
       .attr('class', 'pipeline-node')
+      .attr('data-heap-event', (node) => `clicked.graph.${node.type}`)
```

Comment:
Similar to my comment for the `visible` heap events, we would need to add tests for each type of node to ensure the right heap event gets sent.